content/docs/manage/api-keys.md

```diff
@@ -0,0 +1,77 @@
+---
+title: API Keys
+enableTableOfContents: true
+redirectFrom:
+  - /docs/get-started-with-neon/using-api-keys
+---
+
+Any action performed in the Neon Console can be performed using the [Neon API](https://neon.tech/api-reference). Using the Neon API requires an API key. This topic describes how to generate, revoke, and use API keys.
```

Comment:
```suggestion
You can manage your projects, branches, databases, and users using the [Neon API](https://neon.tech/api-reference). Using the Neon API requires an API key. This topic describes how to generate, revoke, and use API keys.
```

I'm not sure of using the `Any action performed`. You can't do things like passwordless login or create an API key

Reply:
OK. I will change that to "most actions".

Reply:
Sounds good